rtgui/filecatalog.cc

```diff
@@ -1095,41 +1095,41 @@ void FileCatalog::developRequested (std::vector<FileBrowserEntry*> tbe, bool fas
             // and also those which effect is not pronounced after reducing the image size
             // TODO!!! could expose selections below via preferences
             if (fastmode) {
-                if (options.fastexport_bypass_sharpening         ) {
-                    params.sharpening.enabled          = false;
-                }
+                if (!options.fastexport_use_fast_pipeline) {
+                    if (options.fastexport_bypass_sharpening         ) {
```

Comment:
If you stumble upon crazy whitespace, remove it! It might have made sense when first conceived, but tends to rot on every indenter run. Don't miss out to look above and below, if old code hurts the eyes. :wink: